packages/terra-form-textarea/src/orion-fusion-theme/Textarea.module.scss

```diff
@@ -0,0 +1,98 @@
+:local {
+  .orion-fusion-theme {
+    // Default
+    --terra-form-textarea-background-color: #fff;
+    --terra-form-textarea-background-image: none;
+    --terra-form-textarea-background-size: 0 0;
+    --terra-form-textarea-border-color: #dedfe0;
+    --terra-form-textarea-border-radius: 3px;
+    --terra-form-textarea-border-style: solid;
+    --terra-form-textarea-border-width: 1px;
+    --terra-form-textarea-color: #1c1f21;
+    --terra-form-textarea-font-size: 1rem;
+    --terra-form-textarea-line-height: 1.416667;
+    --terra-form-textarea-padding-bottom: 0.25rem;
+    --terra-form-textarea-padding-left: 0.25rem;
+    --terra-form-textarea-padding-right: 0.25rem;
+    --terra-form-textarea-padding-top: 0.25rem;
+
+    // Default - Hover  
+    --terra-form-textarea-hover-background-color: #fff;
+    --terra-form-textarea-hover-border-color: #dedfe0;
+    --terra-form-textarea-hover-border-style: solid;
+    --terra-form-textarea-hover-border-width: 1px;
+    --terra-form-textarea-hover-color: #1c1f21;
+
+    // Default - Focus
+    --terra-form-textarea-focus-background-size: 0 0;
+    --terra-form-textarea-focus-border-color: #4cb2e9;
+    --terra-form-textarea-focus-border-style: solid;
+    --terra-form-textarea-focus-border-width: 1px;
+    --terra-form-textarea-focus-background-color: #fff;
+    --terra-form-textarea-focus-box-shadow: 0 0 8px #a6d9f4;
+    --terra-form-textarea-focus-color: #1c1f21;
+    --terra-form-textarea-focus-opacity: 1;
+    --terra-form-textarea-focus-transition-duration: 0s;
+    --terra-form-textarea-focus-transition-timing-function: ease;
+    --terra-form-textarea-focus-ltr-box-shadow: 0 0 8px #a6d9f4;
+    --terra-form-textarea-focus-rtl-box-shadow: 0 0 8px #a6d9f4;
+
+    // Placeholder
+    --terra-form-textarea-placeholder-color: #767676;
+    --terra-form-textarea-placeholder-font-style: normal;
+
+    // Placeholder - Disabled
+    --terra-form-textarea-placeholder-disabled-color: #c4c6c7;
+    --terra-form-textarea-placeholder-disabled-font-style: normal;
+
+    // Disabled
+    --terra-form-textarea-disabled-background-color: #f4f4f4;
+    --terra-form-textarea-disabled-border-color: #b2b5b6;
+    --terra-form-textarea-disabled-border-style: solid;
+    --terra-form-textarea-disabled-border-width: 1px;
+    --terra-form-textarea-disabled-color: #9b9fa1;
+    --terra-form-textarea-disabled-opacity: 1;
+
+    // Invalid
+    --terra-form-textarea-error-background-image: none;
+    --terra-form-textarea-error-background-color: #fff;
+    --terra-form-textarea-error-border-color: #dedfe0;
+    --terra-form-textarea-error-border-style: solid;
+    --terra-form-textarea-error-border-width: 1px;
+    --terra-form-textarea-error-box-shadow: none;
```

Comment:
It looks like this theme var is used in two places: https://github.com/cerner/terra-core/blob/master/packages/terra-form-textarea/src/Textarea.module.scss#L126

Do you know which other theme var is being used is 2 places? I'm noticing 75 theme vars in theTextarea.module.scss file, but only 73 theme vars in this theme file

Reply:
The other one is `--terra-form-textarea-focus-opacity`.

Reply:
tricky tricky 😄